src/bankAccount.js

```diff
@@ -1,7 +1,27 @@
 'use strict';
 
-function BankAccount() {
-  // Write your code here
+function BankAccount(name, amount) {
+  this.name = name;
+  this.amount = amount;
+  this.accountHistory = [`Initial: ${this.amount}`];
+}
+
+BankAccount.prototype.getInfo = function() {
+  return `Name: ${this.name}, Amount: ${this.amount}$`;
+};
+
+BankAccount.prototype.addMoney = function(salary, operation) {
+  this.amount += salary;
+  this.accountHistory.push(`${operation}: ${salary}`);
+};
+
+BankAccount.prototype.withdrawMoney = function(minusMoney, operation) {
```

Comment:
не привязывайся к самой операции
```suggestion
BankAccount.prototype.withdrawMoney = function(amount, operation) {
```